### DISCUSSION TITLE
Copilot organisation subscription does not work after personal trial ends

### DISCUSSION THREAD

**Discussion by reilem** [community/community#52507] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I have two copilot subscriptions. One personal and one from an organisation. My personal subscription has expired and I now get this message whenever I try to activate copilot in VSCode:

```
[INFO] [auth] [2023-04-11T09:29:38.813Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-04-11T09:29:38.815Z] GitHub Copilot could not connect to server. Extension activation failed: "Thank you for using GitHub Copilot. Your subscription has ended."
```

So it does not work even though I have a valid subscription via an organisation, and this subscription works for colleagues who have tried to use it. I have this same error on each IDE & plugin I try to activate. I have tried looking around all settings and I cannot find any other actions I can take to discard my personal subscription or activate the organisation subscription.

**Comment by Akash1134** [comment]
Hi @reilem , 

Thanks for getting in touch.

Could you please try the following?

- Log out of Copilot from your IDE and log back in to see if that helps.
- If not, install the extension version 1.71.8269 or simply roll back to a previous version of Copilot.

Thanks!

**Reply by alvesvaren** [reply]
I have the same issue. Logging out and back in does not help, same with an older extension version. It says "Your GitHub Copilot subscription has ended." on top of the https://github.com/settings/copilot page, however if I scroll down a bit it says I have it enabled from one of my organizations

**Comment by reilem** [comment]
My organisation was able to solve this eventually. The problem was that they had added me to the organisation but not activated my copilot subscription properly. When they went and explicitly gave me access to copilot, my personal subscription ended, I received a partial refund for the unused days, and I was automatically transferred to the company subscription. So for anyone having this same issue, make sure your organisation activates your copilot subscription properly. Then you should automatically get access to it.
